src/esphome/sensor/mpr121_sensor.h

```diff
@@ -0,0 +1,97 @@
+#ifndef ESPHOME_SENSOR_MPR121_H
+#define ESPHOME_SENSOR_MPR121_H
+
+#include "esphome/defines.h"
+
+#ifdef USE_MPR121
+
+#include "esphome/sensor/sensor.h"
+#include "esphome/binary_sensor/binary_sensor.h"
+#include "esphome/i2c_component.h"
+#include "esphome/component.h"
+
+#define MPR121_I2CADDR_DEFAULT 0x5A
```

Comment:
Do not use defines (discouraged by google C++ guide and good practice)

```suggestion
static const uint8_t MPR121_DEFAULT_I2C_ADDR = 0x5A;
```

Or... I would just remove it entirely (in application.cpp no const is used, so having it here doesn't do anything)